nova/compute/manager.py

```diff
@@ -47,6 +47,7 @@
 import futurist
 from keystoneauth1 import exceptions as keystone_exception
 import os_traits
+from oslo_concurrency.lockutils import _semaphores
```

Comment:
The code in Nova usually consumes `lockutils` like so: `from oslo_concurrency import lockutils`. I'd prefer if we keep it consistent.

Reply:
fixed